### PR TITLE
Jonb/add icons

### PR DESCRIPTION
**Pull request by jonborchardt**
image

### CODE REVIEW

File: demo/src/components/Menu.js [hunk 1]
```diff
@@ -116,3 +116,16 @@ const WrappingLink = styled(InternalLink)`
     white-space: pre-wrap;
     word-wrap: break-word;
 `;
+
+// TODO: this will eventually be exported from varnish... but needed here now
+class SvgIcon extends React.PureComponent {
+  IconImg = styled.img`
+    vertical-align: initial;
+  `;
```
Comment (codeviking) [inline demo/src/components/Menu.js]: This only works in select scenarios (i.e. the parent has to have the right `display` type).

If you're trying to center this, I don't think it's the concern of the icon, but rather the parent layout. I'd propose something like:

```
<FlexRow vAlign={true}>
   <Icon />
   <OtherStuff />
</FlexRow>
```

In other words, I don't know that we want to capture this behavior in the child -- rather, we want to throw it in some sort of parent for greater flexibility, etc.

Comment (jonborchardt) [reply demo/src/components/Menu.js]: the issue is, that the ant parent does a funny thing and adds a negative em to vertical-align...

**Comment by jonborchardt** [comment]
i dont have access to this team city.... @joelgrus can you help me get access or see the logs?

**Comment by joelgrus** [comment]
there should be a "log in as guest" button

but in this case your problem is

```
[19:30:50][Step 1/2] ./src/components/Menu.js
[19:30:50][Step 1/2]   Line 128:  'IconImg' is not defined  react/jsx-no-undef
```

File: demo/src/components/Menu.js [hunk 2]
```diff
@@ -116,3 +116,16 @@ const WrappingLink = styled(InternalLink)`
     white-space: pre-wrap;
     word-wrap: break-word;
 `;
+
+// TODO: this will eventually be exported from varnish... but needed here now
+class SvgIcon extends React.PureComponent {
+  IconImg = styled.img`
```
Comment (joelgrus) [inline demo/src/components/Menu.js]: const IconImg

Comment (jonborchardt) [reply demo/src/components/Menu.js]: fixed

**Pull request merged**